src/site/stages/build/process-cms-exports/tests/transformed-entities/paragraph-collapsible_panel.json

```diff
@@ -8,165 +8,40 @@
     "fieldCollapsiblePanelMulti": false,
     "fieldVaParagraphs": [
       {
-        "uuid": "827bd462-051e-4234-a678-f82ce1743f92",
-        "langcode": [
-          {
-            "value": "en"
-          }
-        ],
-        "type": [
-          {
-            "target_id": "collapsible_panel_item",
-            "target_type": "paragraphs_type",
-            "target_uuid": "e2a6831d-9d06-45d6-940e-dc542830f6ec"
-          }
-        ],
-        "status": [
-          {
-            "value": true
-          }
-        ],
-        "created": [
-          {
-            "value": "2019-03-22T18:44:04+00:00",
-            "format": "Y-m-d\\TH:i:sP"
-          }
-        ],
-        "behaviorSettings": [
-          {
-            "value": []
-          }
-        ],
-        "defaultLangcode": [
-          {
-            "value": true
-          }
-        ],
-        "revisionTranslationAffected": [],
-        "fieldTitle": [
-          {
-            "value": "Mental health resources for Veterans, service members, and families"
-          }
-        ],
-        "fieldVaParagraphs": [],
-        "fieldWysiwyg": [
-          {
-            "value": "<p><strong><a href=\"https://www.militaryonesource.mil/mental-health\">Military OneSource</a></strong><br />\r\nThis free service provides expert support to connect military personnel and their families with the best available resources to fit their needs. For support, visit the Military OneSource website or call <a href=\"tel:+1800-342-9647\">800-342-9647</a> anytime, day or night.</p>\r\n\r\n<p><strong><a href=\"http://pdhealth.mil/\">The Psychological Health Resource Center</a></strong><br />\r\nThe center works to improve the lives of Veterans, service members, and their families by advancing excellence in psychological health care, readiness, and prevention.</p>\r\n",
-            "format": "rich_text"
-          }
-        ],
-        "baseType": "paragraph",
-        "contentModelType": "paragraph-collapsible_panel_item"
+        "contentModelType": "paragraph-collapsible_panel_item",
+        "entity": {
+          "entityType": "paragraph",
+          "entityBundle": "collapsible_panel_item",
+          "fieldTitle": "Mental health resources for Veterans, service members, and families",
+          "fieldVaParagraphs": [],
+          "fieldWysiwyg": {
+            "processed": ""
```

Comment:
This `processed` being empty despite the entity having a value like `<p><strong><a href=\"https://www.militaryonesource.mil/mental-health\">Military OneSourc ...` is suspicious